netlify.toml

```diff
@@ -41,7 +41,7 @@
   NEXT_PUBLIC_INTERCOM_APP_ID = "kn5di10s"
 
 [context.deploy-preview.environment]
-  VITE_SANITY_TAG = "default"
+  VITE_SANITY_TAG = "pinned-projects"
```

Comment:
temporarily set this to a specific graphql endpoint tag so it doesn't interfere with other work
will need to set it back to default before merging

Reply:
don't forget to reverse this and deploy graphql to staging default